### PR TITLE
Add page for job_history view

### PR DESCRIPTION
**Pull request by fabriziomello**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/job_history.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+keywords: [jobs, information]
+tags: [background jobs, scheduled jobs, automation framework, scheduled views]
+api:
+  license: community
+  type: view
+---
+
+## timescaledb_information.job_history
+
+Shows information about failed and successull jobs run by the automation framework.
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
Shows information about the jobs run by the automation framework.
```

File: api/job_history.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+This includes jobs set up for user-defined actions and jobs run by policies
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
This includes jobs set up for user-defined actions, and jobs run by policies
```

File: api/job_history.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+created to manage data retention, continuous aggregates, compression, and
+other automation policies. For more information about automation policies,
+see the [policies][actions] section.
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
see [User-Defined Actions][actions].
```

File: api/job_history.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+### Available columns
+
+|Name|Type|Description|
+|-|-|-|
+|`id`|INTEGER|The sequencial ID to identify the job execution|
+|`job_id`|INTEGER|The ID of the background job created to implement the policy|
+|`succeeded`|BOOLEAN|`TRUE` if the job run without any error, `FALSE` for failed executions|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`succeeded`|BOOLEAN|`TRUE` when the job ran successfully, `FALSE` for failed executions|
```

File: api/job_history.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+|`proc_schema`|TEXT|Schema name of the function or procedure executed by the job|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`proc_schema`|TEXT| The schema name of the function or procedure executed by the job|
```

File: api/job_history.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+|`proc_name`|TEXT|Name of the function or procedure executed by the job|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`proc_name`|TEXT| The name of the function or procedure executed by the job|
```

File: api/job_history.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+|`pid`|INTEGER|The process ID of the background worker executing the job. This is `NULL` in the case of a job crash|
+|`start_time`|TIMESTAMP WITH TIME ZONE|Start time of the job|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`start_time`|TIMESTAMP WITH TIME ZONE| The time the job started|
```

File: api/job_history.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+|`finish_time`|TIMESTAMP WITH TIME ZONE|Time when error was reported|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`finish_time`|TIMESTAMP WITH TIME ZONE| The time when the error was reported|
```

File: api/job_history.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+|`config`|JSONB|Job configuration at the moment of the execution|
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
|`config`|JSONB| The job configuration at the moment of execution|
```

File: api/job_history.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+|`sqlerrcode`|TEXT|The error code associated with this error, if any. See the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/errcodes-appendix.html) for a full list of error codes|
+|`err_message`|TEXT|The detailed error message|
+
+### Sample usage
+
+See information about recent job executions:
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
To retrieve information about recent jobs:
```

File: api/job_history.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+   1001 | 1779408 | public      | custom_job_error | f         |            | 22012      | division by zero
+   1000 | 1779467 | public      | custom_job_ok    | t         | {"foo": 1} |            | 
+   1001 | 1779468 | public      | custom_job_error | f         | {"bar": 1} | 22012      | division by zero
+(5 rows)
+```
+
+### Error retention policy
+
+The informational view `timescaledb_information.job_history` is defined on top
+of the table `_timescaledb_internal.bgw_job_stat_history` in the internal schema. To
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
of the `_timescaledb_internal.bgw_job_stat_history` table in the internal schema. To
```

File: api/job_history.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+   1000 | 1779407 | public      | custom_job_ok    | t         |            |            | 
+   1001 | 1779408 | public      | custom_job_error | f         |            | 22012      | division by zero
+   1000 | 1779467 | public      | custom_job_ok    | t         | {"foo": 1} |            | 
+   1001 | 1779468 | public      | custom_job_error | f         | {"bar": 1} | 22012      | division by zero
+(5 rows)
+```
+
+### Error retention policy
+
+The informational view `timescaledb_information.job_history` is defined on top
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
The `timescaledb_information.job_history` informational view is defined on top
```

File: api/job_history.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+of the table `_timescaledb_internal.bgw_job_stat_history` in the internal schema. To
+prevent this table from growing too large, a system background job
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
prevent this table from growing too large, the
```

File: api/job_history.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+`Job History Log Retention Policy [3]` is enabled by default,
```
Comment (billy-the-fish) [inline api/job_history.md]:
```suggestion
`Job History Log Retention Policy [3]` system background job is enabled by default,
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Great content, awesome. A few language suggestions for you.

File: api/job_history.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+created to manage data retention, continuous aggregates, compression, and
+other automation policies. For more information about automation policies,
+see the [policies][actions] section.
+
+### Available columns
+
+|Name|Type|Description|
+|-|-|-|
+|`id`|INTEGER|The sequencial ID to identify the job execution|
+|`job_id`|INTEGER|The ID of the background job created to implement the policy|
```
Comment (fabriziomello) [inline api/job_history.md:24]: Looks like there's no change or am I missing something?

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**